build-config/scripts/onie-mk-iso.sh

```diff
@@ -71,17 +73,22 @@ RECOVERY_ISO_IMAGE=${11}
     exit 1
 }
 if [ "$UEFI_ENABLE" = "yes" ] ; then
-    [ -r "${GRUB_HOST_LIB_UEFI_DIR}/efinet.mod" ] || {
-        echo "ERROR: Does not look like valid GRUB x86_64-efi library directory: $GRUB_HOST_LIB_UEFI_DIR"
+    [ -r "${GRUB_TARGET_LIB_UEFI_DIR}/efinet.mod" ] || {
+        echo "ERROR: Does not look like valid GRUB ${ARCH}-efi library directory: $GRUB_TARGET_LIB_UEFI_DIR"
         exit 1
     }
     [ -x "${GRUB_HOST_BIN_UEFI_DIR}/grub-mkimage" ] || {
-        echo "ERROR: Does not look like valid GRUB x86_64-efi bin directory: $GRUB_HOST_BIN_UEFI_DIR"
+        echo "ERROR: Does not look like valid GRUB ${ARCH}-efi bin directory: $GRUB_HOST_BIN_UEFI_DIR"
```

Comment:
grub-mkimage is standard utility. (http://gittup.org/cgi-bin/man/man2html?1+grub-mkimage)